src/components/Reports/ReportsFilters/ReportsFilters.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+const ReportsFilters = ({ domains, domainSelected, pages, changeDomain }) => {
+  return (
+    <header className="report-filters">
+      <div className="report-wrapper">
+        <h3>
+          <FormattedMessage id="reports_filters.title" />
+        </h3>
+        <p>
+          <FormattedMessage id="reports_filters.description" />
+        </p>
+        <form action="#" className="form-filters">
+          <fieldset className="filter">
+            <label htmlFor="domain">
+              <FormattedMessage id="reports_filters.domain" />
+            </label>
+            <span className="dropdown-arrow" />
+            <select
+              onChange={changeDomain}
+              id="domain"
+              className=""
+              name=""
+              disabled={domains.length === 0}
```

Comment:
Since outside you are writing `if (domain.length)`, what do you think about this:
```suggestion
              disabled={!domains.length}
```

Reply:
fixed